main.tf

```diff
@@ -3,30 +3,30 @@ locals {
     for index, rule in var.rules :
     merge(rule, {
       "name" = index
-      "Name" = "${replace(index, "_", "-")}-rule"
+      "Name" = var.append_rule_postfix ? "${replace(index, "_", "-")}-rule" : index
     })
   ])
   eventbridge_targets = flatten([
     for index, rule in var.rules : [
       for target in var.targets[index] :
       merge(target, {
         "rule" = index
-        "Name" = "${replace(index, "_", "-")}-rule"
+        "Name" = var.append_rule_postfix ? "${replace(index, "_", "-")}-rule" : index
       })
     ] if length(var.targets) != 0
   ])
   eventbridge_connections = flatten([
     for index, conn in var.connections :
     merge(conn, {
       "name" = index
-      "Name" = "${replace(index, "_", "-")}-connection"
+      "Name" = var.append_rule_postfix ? "${replace(index, "_", "-")}-rule" : index
```

Comment:
We don't use this right? If so, we can delete it from here and keep the original implementation.